src/sinks/elasticsearch.rs

```diff
@@ -64,8 +64,15 @@ pub struct ElasticSearchConfig {
 
     pub aws: Option<RegionOrEndpoint>,
     pub tls: Option<TlsOptions>,
-    #[serde(default)]
-    pub bulk_action: BulkAction,
+    pub bulk_action: Option<String>,
+}
+
+impl ElasticSearchConfig {
+    fn bulk_action(&self) -> crate::Result<Template> {
+        let value = self.bulk_action.as_deref().unwrap_or("index");
+        let value = Template::try_from(value).context(BatchActionTemplate)?;
+        Ok(value)
```

Comment:
```suggestion
        Template::try_from(value).context(BatchActionTemplate)
```

The smallest of nits.

Reply:
Doesn't seem to [be working](https://github.com/timberio/vector/pull/7805/checks?check_run_id=2792136224#step:9:19) 🤔

Reply:
That link isn't showing me anything, but I'm guessing it's because the error needs to be mapped. You could maybe do:

```rust
        Template::try_from(value).context(BatchActionTemplate).map_err(Into::into)
```

Not a big deal though, what you have is fine.